<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1038</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HQ Battery</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#statusbar{
	background-color: rgb(36, 39, 46);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color:#0f1228;
color: rgb(255, 255, 255);
}
QLabel{
	color: rgb(255, 255, 255);
}
QFrame[frameShape=&quot;4&quot;] {
	background-color: rgb(23, 54, 133);
}</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{
border-image: url(:/image/top_title.png);
}
#label_logo{
	border-image: url(:/image/logo.png);
}
#label_company{	
	font: 700 20pt &quot;Microsoft YaHei UI&quot;;
}
#label_current_time{
	font: 700 14pt &quot;Microsoft YaHei UI&quot;;
}</string>
    </property>
    <widget class="QLabel" name="label_logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_company">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>徐州华强电器制造有限公司</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>1849</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/image/close.png);
}
QPushButton:hover{
	border-image: url(:/image/close_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_max">
     <property name="geometry">
      <rect>
       <x>1790</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/image/max.png);
}
QPushButton:hover{
	border-image: url(:/image/max_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_min">
     <property name="geometry">
      <rect>
       <x>1730</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	border-image: url(:/image/min.png);
}
QPushButton:hover{
	border-image: url(:/image/min_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_time">
     <property name="geometry">
      <rect>
       <x>1430</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2025-05-29 08:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_user">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>未登录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_history">
     <property name="geometry">
      <rect>
       <x>1630</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/btn_history.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_settings">
     <property name="geometry">
      <rect>
       <x>1680</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/设置.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1911</width>
      <height>951</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="LeftStatsForm" name="widget_left" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>931</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_center" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>1401</width>
        <height>921</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_center{
border-image: url(:/image/主页_中间.png);
}
QLabel{
	font: 700 10pt &quot;Microsoft YaHei UI&quot;;
}</string>
      </property>
     </widget>
     <widget class="RightStatsForm" name="widget_right" native="true">
      <property name="geometry">
       <rect>
        <x>1670</x>
        <y>0</y>
        <width>231</width>
        <height>931</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>widget_2</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeftStatsForm</class>
   <extends>QWidget</extends>
   <header>leftstatsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RightStatsForm</class>
   <extends>QWidget</extends>
   <header>rightstatsform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
